cmd/restic/cmd_prune.go

```diff
@@ -2,17 +2,23 @@ package main
 
 import (
 	"fmt"
+	"sort"
 	"time"
 
+	"golang.org/x/sync/errgroup"
+
 	"github.com/restic/restic/internal/debug"
 	"github.com/restic/restic/internal/errors"
-	"github.com/restic/restic/internal/index"
+	"github.com/restic/restic/internal/pack"
 	"github.com/restic/restic/internal/repository"
 	"github.com/restic/restic/internal/restic"
 
 	"github.com/spf13/cobra"
 )
 
+var errorIndexIncomplete = errors.New("Error: Index is not complete!")
+var errorDataFilesMissing = errors.New("Error: Data files from index missing in repo!")
```

Comment:
I don't like error messages shouting. The existing ones don't do this, either.

Reply:
The convention for error messages in Go is to start with a lower-case letter.

Reply:
And the "Error:" should also not be necessary.

Reply:
is changed.